packages/cubejs-server-core/src/core/types.ts

```diff
@@ -227,6 +227,7 @@ export interface CreateOptions {
   serverless?: boolean;
   allowNodeRequire?: boolean;
   semanticLayerSync?: (context: RequestContext) => Promise<BiToolSyncConfig[]> | BiToolSyncConfig[];
+  fastReloadEnabled?: boolean;
```

Comment:
We don't use any prefixes/suffixes for `boolean` options. I belive it should `fastReload`